src/api/SFA.DAS.Provider.Events.Api/NLog.config

```diff
@@ -1,45 +1,16 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-      xsi:schemaLocation="http://www.nlog-project.org/schemas/NLog.xsd NLog.xsd"
-      autoReload="true"
-      throwExceptions="false"
-      internalLogLevel="info" internalLogFile="internal-nlog.txt">
-
+      xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">
   <extensions>
     <add assembly="SFA.DAS.NLog.Targets.Redis"/>
   </extensions>
 
-  <!--
-  See https://github.com/nlog/nlog/wiki/Configuration-file
-  for information on customizing logging rules and outputs.
-   -->
   <targets>
-
-    <!--
-    add your targets here
-    See https://github.com/nlog/NLog/wiki/Targets for possible targets.
-    See https://github.com/nlog/NLog/wiki/Layout-Renderers for the possible layout renderers.
-    -->
-
-    <target xsi:type="Redis" name="redis" connectionStringName="Redis" environmentKeyName="EnvironmentName" appName="das-providerevents-api" includeAllProperties="true" layout="${message}">
-      <field name="requestId" layout="${aspnet-traceidentifier}"/>
-      <field name="loggerTitle" layout="${logger}"/>
-    </target>
-
-    <!--
-    Write events to a file with the date in the filename.
-    <target xsi:type="File" name="f" fileName="${basedir}/logs/${shortdate}.log"
-            layout="${longdate} ${uppercase:${level}} ${message}" />
-    -->
+    <target xsi:type="Redis" name="Redis" connectionStringName="LoggingConnectionString" environmentKeyName="EnvironmentName" appName="das-providerevents-api" includeAllProperties="true" layout="${message}"></target>
   </targets>
 
   <rules>
-    <!-- add your logging rules here -->
-    <logger name="*" minlevel="Trace" writeTo="redis" />
-    <!--
-    Write all events with minimal level of Debug (So Debug, Info, Warn, Error and Fatal, but not Trace)  to "f"
-    <logger name="*" minlevel="Debug" writeTo="f" />
-    -->
+    <logger name="*" minlevel="Debug" writeTo="Redis" />
```

Comment:
Debug level isn't suitable for releases

Reply:
Currently Debug in the das-dotnet-template, e.g. https://github.com/SkillsFundingAgency/das-dotnet-templates/blob/bd0a77544d4ddeedd716eebbe2cc0bcf1378b604/src/working/templates/Web/src/SFA.DAS.WebTemplateSourceName.Web/nlog.config#L18 so resolving for now.